docs/upgrade/0.3.x_to_0.4.rst

```diff
@@ -0,0 +1,94 @@
+Upgrade from 0.3.x to 0.4
+=========================
+
+SecureDrop 0.4 requires the use of Tails 3. It includes substantial
+changes to the Admin tooling used for managing the configuration
+for the Application and Monitor Servers, and modifies the location
+of the configuration on Admin Workstation to prevent conflicts
+in the future.
+
+.. _0.4-upgrade-procedure:
+
+  .. note::
+    All Admin and Journalist Workstations must be upgraded to Tails 3 for use
+    with SecureDrop 0.4. Follow the :doc:`/upgrade_to_tails_3x` guide for
+    detailed instructions on upgrading if you have not already done so.
+
+The steps below should be performed on **both the Admin* and all Journalist
```

Comment:
The asterisk should _not_ be there, it was left over from when I tried to italicize the workstation titles within the bolded text and couldn't get it right—will remove.